yb_migrate/src/migration/oracle.go

```diff
@@ -341,13 +336,13 @@ func OracleGetAllPartitionNames(source *utils.Source, tableName string) []string
 		var partitionName string
 		err = rows.Scan(&partitionName)
 		if err != nil {
-			errMsg := fmt.Sprintf("error in scanning query rows: %v\n", err)
-			utils.ErrExit(errMsg)
+			utils.ErrExit("error in scanning query rows: %v", err)
 		}
 		partitionNames = append(partitionNames, partitionName)
 
 		// TODO: Support subpartition(find subparititions for each partition)
 	}
 
+	log.Infof("Partition Names:%q\n", partitionNames)
```

Comment:
Log parent table name also with the partition name.
Something like this:
```
log.Infof("Partition Names for parent table %q: %q", partitionNames)
```

Aslo, I think "\n" with log.Infof() and log.Info() are not required. Please check and remove if the case

Reply:
There is an explicit log.Infoln() which appends "\n", the helper functions that we have written (utils.PrintandLog(), utils.ErrExit()) have an implicit newline appended